app/javascript/dashboard/components-next/message/bubbles/BaseAttachment.vue

```diff
@@ -40,7 +40,7 @@ const senderName = computed(() => {
             <Icon :icon="icon" class="text-white size-4" />
           </slot>
         </div>
-        <div class="space-y-1">
+        <div class="space-y-1 truncate">
```

Comment:
The truncate class is already added below for the text wrapper
```suggestion
        <div class="space-y-1 overflow-hidden">
```